packages/flutter_tts/tizen/src/flutter_tts_tizen_plugin.cc

```diff
@@ -199,6 +203,28 @@ class FlutterTtsTizenPlugin : public flutter::Plugin {
     SendResult(flutter::EncodableValue(list));
   }
 
+  void OnGetDefaultVoice() {
+    std::optional<std::string> voice = tts_->GetDefaultVoice();
+    if (!voice.has_value()) {
+      result_->Error("Operation failed",
+                     "Failed to get TTS engine's default voice.");
+      result_ = nullptr;
+      return;
+    }
+    SendResult(flutter::EncodableValue(*voice));
+  }
+
+  void OnGetMaxSpeechInputLength() {
+    std::optional<int32_t> length = tts_->GetMaxSpeechInputLength();
+    if (!length.has_value()) {
+      result_->Error("Operation failed",
+                     "Failed to get TTS engine's max speech input length.");
+      result_ = nullptr;
```

Comment:
`getMaxSpeechInputLength` is supposed to return a nullable integer, so you could just return a null value in case of failure.

Reply:
done